seacrowd/sea_datasets/saltik/saltik.py

```diff
@@ -0,0 +1,165 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Saltik is a dataset for benchmarking non-word error correction method accuracy in evaluating Indonesian words.
+It consists of 58,532 non-word errors generated from 3,000 of the most popular Indonesian words.
+"""
+import os
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+import jsonlines
+import json
+
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Tasks, Licenses
+
+# TODO: Add BibTeX citation
+_CITATION = """\
+@article{,
+  author    = {Audah, Hanif Arkan and Yuliawati, Arlisa and Alfina, Ika},
+  title     = {A Comparison Between SymSpell and a Combination of Damerau-Levenshtein Distance With the Trie Data Structure},
+  journal   = {2023 10th International Conference on Advanced Informatics: Concept, Theory and Application (ICAICTA)},
+  volume    = {},
+  year      = {2023},
+  url       = {https://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=10390399&casa_token=HtJUCIGGlWYAAAAA:q8ll1RWmpHtSAq2Qp5uQAE1NJETx7tUYFZIvTO1IWoaYy4eqFETSsm9p6C7tJwLZBGq5y8zc3A&tag=1},
+  doi       = {},
+  biburl    = {https://scholar.googleusercontent.com/scholar.bib?q=info:5C2ZSuuJCDcJ:scholar.google.com/&output=citation&scisdr=ClF7-LnEEO-epgRychQ:AFWwaeYAAAAAZa50ahRiC4_sEsA-4WL6HB3SpIo&scisig=AFWwaeYAAAAAZa50aiWPCHL1NAShFbo-QupEL2c&scisf=4&ct=citation&cd=-1&hl=en},
+  bibsource = {google}
+}
+"""
```

Comment:
for the citation, we can copy directly from the github: https://github.com/ir-nlp-csui/saltik?tab=readme-ov-file#references

Reply:
ok changed